engine/src/main/java/org/terasology/rendering/nui/widgets/UISlider.java

```diff
@@ -56,6 +58,9 @@ public boolean onMouseClick(NUIMouseClickEvent event) {
         @Override
         public void onMouseRelease(NUIMouseReleaseEvent event) {
             active = false;
+            if (uiSliderOnChangeListener != null) {
+                uiSliderOnChangeListener.onSliderMouseDown(getValue());
```

Comment:
This naming seems confusing. `onMouseRelease` seems like the right phase, but it is calling an _onDown_ method? Shouldn't the listener method be named similarly? `onMouseRelease` as well or `onChangeTriggered` perhaps.